README.md

```diff
@@ -85,56 +85,56 @@ Create specified in ScConstruction elements.
 
     const construction = new ScConstruction();
 
-    construction.createNode(ScType.NodeConst, myNode);
-    construction.createLink(
+    construction.generateNode(ScType.NodeConst, myNode);
+    construction.generateLink(
       ScType.LinkConst,
       new ScLinkContent(linkContent, ScLinkContentType.String),
       myLink
     );
-    construction.createEdge(
+    construction.generateConnector(
       ScType.EdgeAccessConstPosPerm,
       myNode,
       fakeNodeAddr
     );
 
-    const res = await client.createElements(construction);
+    const res = await client.generateElements(construction);
 ```
 
-## `client.createElementsBySCs(scsText: string[] | ISCs[]): boolean[]`
+## `client.generateElementsBySCs(scsText: string[] | ISCs[]): boolean[]`
 
 Create specified in ScConstruction elements by SCs text and puts them in structure. Returned boolean represents whether SCs text processing was 
-successfull.
+successful.
 
 ```ts
     import { client } from "../path-to-client";
 
-    const res = await client.createElementsBySCs(
+    const res = await client.generateElementsBySCs(
         ["my_class -> node1;;", "node1 => relation: node2;;"]
     );
 
-    const res2 = await client.createElementsBySCs(
+    const res2 = await client.generateElementsBySCs(
         [{scs: "my_class -> node1;;", output_structure: new ScAddr(0)}]
     );
 ```
 
-## `client.deleteElements(addrs: ScAddr[]): boolean`
+## `client.eraseElements(addrs: ScAddr[]): boolean`
 
-Delete specified elements. Returned boolean represents whether deleting was successfull.
+Delete specified elements. Returned boolean represents whether deleting was successful.
 
 ```ts
     import { client } from "../path-to-client";
 
-    const res = await client.deleteElements([addr1, addr2]);
+    const res = await client.eraseElements([addr1, addr2]);
 ```
 
-## `client.checkElements(addrs: ScAddr[]): ScType[]`
+## `client.getElementTypes(addrs: ScAddr[]): ScType[]`
```

Comment:
this name gives vibes that element has several types, what about getElementsTypes?

Reply:
It is incorrect for English

Reply:
Okay